source/backend/database/models/savedRecipesModel.js

```diff
@@ -17,11 +17,12 @@ const db = require('../dbConfig');
     
 /**
  * delete savedRecipe record by recipeId
+ * @param userId
  * @param savedRecipeId
  */
- async function removeById(savedRecipeId) {
-    const result = await db('recipes') 
-      .where({ savedRecipeId })
+ async function removeSavedRecipe(userId, savedRecipeId) {
```

Comment:
We are not really doing a good job following the convention of: verbByField naming convention for those model functions. 

For this one it's best to name it `deleteByUserIdAndSavedRecipeId`.

Reply:
Sorry about that! Fixed.